CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 Changes in this release:
 - Log a warning when the id_attribute of a resource is called id.
 - Ignore `__pycache__` dirs when copying the current module to the test project dir
-- Deprecate setting pip indexes through `--module-repo` option with type `package` in favour of newly introduced `--pip-index-url` option.
+- [inmanta-core > iso7] Deprecate setting pip indexes through `--module-repo` option with type `package` in favour of newly introduced `--pip-index-url` option.
```

Comment:
I'm not very satisfied with this changelog message, but technically speaking nothing has been added for iso6

Reply:
Perhaps it reads better if we invert it? "Added --pip-index-url to set corresponding project config section for inmanta-core 9.0"?